mysql-test/r/mysqld--help-notwin.result

```diff
@@ -2034,7 +2037,6 @@ rpl-skip-tx-api FALSE
 rpl-stop-slave-timeout 604800
 safe-user-create FALSE
 secure-auth TRUE
-secure-file-priv (No default value)
```

Comment:
this line shoudn't be removed as part of this diff.  Looks like @anirbanr-fb forgot to re-record this test in https://github.com/facebook/mysql-5.6/commit/04a32957d785e4546535c754cef9f9768e5c9d0f#diff-0f69f98e011b11eac33c99987aa19fb8.  I'll make a commit removing it which you can rebase your changes on top of.